robot_smach_states/src/robot_smach_states/manipulation/manipulation.py

```diff
@@ -262,80 +277,88 @@ def __init__(self, side, x, y, z, roll=0, pitch=0, yaw=0, time_out=20, pre_grasp
         self.x = x
         self.y = y
         self.z = z
-        self.roll  = roll
+        self.roll = roll
         self.pitch = pitch
-        self.yaw   = yaw
-        self.time_out   = time_out
+        self.yaw = yaw
+        self.time_out = time_out
         self.pre_grasp = pre_grasp
-        self.frame_id  = frame_id
+        self.frame_id = frame_id
         self.delta = delta
 
     def execute(self, userdata=None):
         if not self.delta:
             rospy.logwarn("Transforming to baselink, should become obsolete but this is not yet the case")
-            #target_position_bl = transformations.tf_transform(target_position, "/map","/base_link", tf_listener=self.robot.tf_listener)
             if self.side.send_goal(self.x, self.y, self.z, self.roll, self.pitch, self.yaw,
-                timeout=self.time_out,
-                pre_grasp=self.pre_grasp,
-                frame_id=self.frame_id):
+                                   timeout=self.time_out,
+                                   pre_grasp=self.pre_grasp,
+                                   frame_id=self.frame_id):
                 return 'succeeded'
             else:
                 return 'failed'
         else:
             if self.side.send_delta_goal(self.x, self.y, self.z, self.roll, self.pitch, self.yaw,
-                timeout=self.time_out,
-                pre_grasp=self.pre_grasp,
-                frame_id=self.frame_id):
+                                         timeout=self.time_out,
+                                         pre_grasp=self.pre_grasp,
+                                         frame_id=self.frame_id):
                 return 'succeeded'
             else:
                 return 'failed'
 
+
 class ArmToQueryPoint(smach.State):
     def __init__(self, robot, side, point_designator, time_out=20, pre_grasp=False, first_joint_pos_only=False):
         """Move the arm to de designated point
         @param point_designator resolves to a PointStamped the gripper should move to"""
-        smach.State.__init__(self, outcomes=['succeeded','failed'])
-        self.robot                = robot
-        self.side                 = side
-        self.point_designator     = point_designator
-        self.time_out             = time_out
-        self.pre_grasp            = pre_grasp
+        smach.State.__init__(self, outcomes=['succeeded', 'failed'])
+        self.robot = robot
+        self.side = side
+        self.point_designator = point_designator
+        self.time_out = time_out
+        self.pre_grasp = pre_grasp
         self.first_joint_pos_only = first_joint_pos_only
```

Comment:
I believe the entire ArmToQueryPoint class is deprecated. It is not used in any of the challenges and it still uses the deprecated amigo API. Should this function be updated or removed entirely?

Reply:
Removed